pyanaconda/geoloc.py

```diff
@@ -338,12 +334,13 @@ def __init__(self,
                             once the class is initialized
         :type refresh_now: bool
         """
-        self._provider = _get_provider(provider_id)
+        provider = AVAILABLE_GEOLOCATION_PROVIDERS.get(provider_id, FedoraGeoIPProvider)
```

Comment:
This looks really weird to me. Everything is with capital letters but you are calling `get` with two parameters. Could you please change this to be less confusing for using?